js/index.js

```diff
@@ -1,7 +1,41 @@
 // Iteration 1: Names and Input
+let hacker1 = "Emma";
+console.log(`The driver's name is ${hacker1}`);
 
+let hacker2 = "Delia";
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+  }
+  else if (hacker1.length < hacker2.length){ 
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters!`);
+    }
+    else {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);
+  }
 
+  // Iteration 3: Loops
+ //not sure how to only print the name only once with a loop?
+for (let i = 0; i < hacker1.length; i++)
+console.log(hacker1.toUpperCase().split("").join(" "));
 
-// Iteration 3: Loops
\ No newline at end of file
+let reverseCase = hacker2.split("").reverse().join("");
+console.log(reverseCase);
+
+if(hacker1.localeCompare(hacker2) === -1){
```

Comment:
the output of localCompare can vary across browsers. Please check the localeCompare documentation on MDN in order to find out what would be a better solution. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare